12/umbraco-cms/reference/templating/macros/partial-view-macros.md

```diff
@@ -15,7 +15,7 @@ All Partial View Macro views inherit from `Umbraco.Cms.Web.Common.Macros.Partial
 @inherits Umbraco.Cms.Web.Common.Macros.PartialViewMacroPage
 ```
 
-The model type for a Partial View Macro is `Umbraco.Cms.Core.Models.PartialViewMacroModel` which contains all of the properties you need to render out content alongwith some additional
+The model type for a Partial View Macro is `Umbraco.Cms.Core.Models.PartialViewMacroModel`. This contains all of the properties you need to render out content alongside some additional
```

Comment:
to keep the docs consistent, can you please make this change to v10 and v11 as well?